### PR TITLE
add licenses for content

### PR DESCRIPTION
**Pull request by escattone**
@chrisdavidmills @peterbe This is my first crack at a `LICENSE.md` file for this content repo. It would apply to the upcoming `archived-content` and `translated-content` repos as well.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,239 @@
+# Licenses for MDN Content
+
+MDN's content is entirely available under various open source licenses.
```
Comment (chrisdavidmills) [inline LICENSE.md:3]: Maybe say "MDN's content (including prose and code examples)..."

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+This section covers the types of content we provide and what licenses
+are in effect for each.
+
+## License for All Documents
```
Comment (chrisdavidmills) [inline LICENSE.md:7]: "License for all prose content" ?

File: LICENSE.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+https://creativecommons.org/licenses/by-sa/2.5/
+
```
Comment (chrisdavidmills) [inline LICENSE.md:10]: For consistency with below sections, add something here like "Prose content is all available under 
([CC-BY-SA 2.5](https://creativecommons.org/licenses/by-sa/2.5/))."

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+Creative Commons is not a party to this License, and makes no warranty whatsoever in connection with the Work. Creative Commons will not be liable to You or any party on any legal theory for any damages whatsoever, including without limitation any general, special, incidental or consequential damages arising in connection to this license. Notwithstanding the foregoing two (2) sentences, if Creative Commons has expressly identified itself as the Licensor hereunder, it shall have all rights and obligations of Licensor.
+
+Except for the limited purpose of indicating to the public that the Work is licensed under the CCPL, neither party will use the trademark "Creative Commons" or any related trademark or logo of Creative Commons without the prior written consent of Creative Commons. Any permitted use will be in compliance with Creative Commons' then-current trademark usage guidelines, as may be published on its website or otherwise made available upon request from time to time.
+
+Creative Commons may be contacted at http://creativecommons.org/.
+```
+
+## Licenses for Code Samples and Snippets within Documents
```
Comment (chrisdavidmills) [inline LICENSE.md:69]: Use sentence case for all titles, so only a capital letter at the start of the sentence.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@escattone just a few small comments here, nothing massive.

**Comment by escattone** [comment]
@chrisdavidmills Thank you for the review! I've incorporated all of your feedback and it's ready for another look whenever your ready.

**Review by chrisdavidmills — APPROVED** [review]
@escattone looks awesome, thanks!

**Pull request merged**